images/base-app/Dockerfile

```diff
@@ -24,7 +24,11 @@ COPY --chmod=777 scripts/startup.sh /opt/gow/startup.sh
 # _does_ handle making the actual drivers available, it doesn't install the ICD
 # JSON.  So we provide one ourselves.  This is harmless on non-NVIDIA systems,
 # so we just do it unconditionally.
+# NOTE: as of nvidia-container-toolkit v1.12, this is no longer necessary. once
```

Comment:
Is it better to create an issue for this for future work?

Reply:
probably :grin:

Reply:
Mind creating one when this is approved :p